packages/components/src/presentation-components/instance-filter-builder/InstanceFilterBuilder.tsx

```diff
@@ -107,18 +107,18 @@ export function usePresentationInstanceFilteringProps(
   initialClasses?: ClassInfo[],
 ): Required<
   Pick<
-    InstanceFilterBuilderProps,
-    | "properties"
-    | "classes"
-    | "selectedClasses"
-    | "onClassSelected"
-    | "onClassDeselected"
-    | "onClearClasses"
-    | "propertyRenderer"
-    | "onRulePropertySelected"
-    | "isDisabled"
+  InstanceFilterBuilderProps,
```

Comment:
Do we know why indent got changed here? I like the previous style better.

Reply:
It is from `@typescript-eslint` that is being used by itwin/eslint-plugin

Reply:
This is from the prettier issue I have explained below, solved now with no change needed